cmd/baton-fullstory/config.go

```diff
@@ -1,30 +1,9 @@
 package main
 
 import (
-	"context"
-
-	"github.com/conductorone/baton-sdk/pkg/cli"
-	"github.com/spf13/cobra"
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/status"
+	"github.com/conductorone/baton-sdk/pkg/field"
 )
 
-// config defines the external configuration required for the connector to run.
-type config struct {
-	cli.BaseConfig `mapstructure:",squash"` // Puts the base config options in the same place as the connector options
-
-	APIKey string `mapstructure:"api-key"`
-}
-
-// validateConfig is run after the configuration is loaded, and should return an error if it isn't valid.
-func validateConfig(ctx context.Context, cfg *config) error {
-	if cfg.APIKey == "" {
-		return status.Error(codes.InvalidArgument, "api token must be provided, use --help for more information")
-	}
-
-	return nil
-}
-
-func cmdFlags(cmd *cobra.Command) {
-	cmd.PersistentFlags().String("api-key", "", "FullStory API Key to authenticate with. ($BATON_API_KEY)")
-}
+var (
+	apikey = field.StringField("api-key", field.WithRequired(true))
```

Comment:
- Add the description `FullStory API Key to authenticate with`.
- Update README
- In `main.go`, add 
 ```go
err := field.Validate(Configuration, v)
	if err != nil {
		return nil, err
	}
```

Reply:
![imagen](https://github.com/user-attachments/assets/b26da7d4-b435-4930-b12c-1e0a8572dc2d)

No need to add `field.Validate` in main.go tho

Reply:
Oh cool I hadn't realized `field.Validate()` is automatically called.